pkg/monitor/monitor.go

```diff
@@ -69,18 +88,35 @@ func (m *Monitor) checkState(p *Process, wg *sync.WaitGroup) {
 	defer wg.Done()
 	switch p.state {
 	case neverStarted:
-		log.Infof("starting %s for the first time", p.Name)
+		p.logger.Printf("starting %s for the first time", p.Name)
+		if p.Path == SyncthingBin {
+			if err := setup.Setup(m.syncthingConfig.secret, m.syncthingConfig.config); err != nil {
+				p.logger.Errorf("failed to setup %s: %w", p.Name, err)
```

Comment:
nit: if you run `make lint` you can see the 3 lint errors on the `%w` directive:

```
Errorf does not support error-wrapping directive %w (govet)
```